build/ci/github-bot/main.go

```diff
@@ -129,6 +132,22 @@ func (g *githubNotifier) SendNotification(ctx context.Context, build *cloudbuild
 	return nil
 }
 
+func (g *githubNotifier) clearBotComments(ctx context.Context, pr int) error {
+	comments, _, err := g.client.Issues.ListComments(ctx, owner, repo, pr, &github.IssueListCommentsOptions{})
+	if err != nil {
+		log.Errorf("Error retriving comment history: %v on PR with id: %s", err, pr)
```

Comment:
Small spelling error: 
```suggestion
		log.Errorf("Error retrieving comment history: %v on PR with id: %s", err, pr)
```